{"kind":"symbol","hierarchy":{"paths":[["doc:\/\/AsyncMoya\/documentation\/AsyncMoya","doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError"]]},"metadata":{"title":"DataError.errorData(_:)","roleHeading":"Case","symbolKind":"case","role":"symbol","modules":[{"name":"AsyncMoya"}],"externalID":"s:9AsyncMoya9DataErrorO05errorC0yACSScACmF","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorData","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}]},"variants":[{"paths":["\/documentation\/asyncmoya\/dataerror\/errordata(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError\/errorData(_:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"서버로부터 받은 오류 메시지 문자열을 직접 저장할 때 사용합니다."}],"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorData","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":")","kind":"text"}],"languages":["swift"],"platforms":["macOS"]}]}],"sections":[],"schemaVersion":{"minor":3,"patch":0,"major":0},"references":{"doc://AsyncMoya/documentation/AsyncMoya":{"title":"AsyncMoya","kind":"symbol","url":"\/documentation\/asyncmoya","abstract":[{"type":"text","text":"Moya를 위한 현대적인 async\/await, Combine, RxSwift 확장 라이브러리"}],"type":"topic","role":"collection","identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya"},"doc://AsyncMoya/documentation/AsyncMoya/DataError":{"title":"DataError","type":"topic","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"DataError","kind":"identifier"}],"url":"\/documentation\/asyncmoya\/dataerror","kind":"symbol","abstract":[{"type":"text","text":"네트워크 요청 및 응답 처리 중 발생할 수 있는 다양한 오류를 표현하는 열거형"}],"navigatorTitle":[{"text":"DataError","kind":"identifier"}],"identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError","role":"symbol"},"doc://AsyncMoya/documentation/AsyncMoya/DataError/errorData(_:)":{"url":"\/documentation\/asyncmoya\/dataerror\/errordata(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorData"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}],"type":"topic","abstract":[{"type":"text","text":"서버로부터 받은 오류 메시지 문자열을 직접 저장할 때 사용합니다."}],"kind":"symbol","title":"DataError.errorData(_:)","identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError\/errorData(_:)","role":"symbol"}}}